sdk/vision/imageAnalysis/samples/typescript/analyzeImageFromLocalFile.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as fs from 'fs';
+import createImageAnalysisClient, { ImageAnalysisClient, ImageAnalysisResultOutput } from '@azure/imageAnalysis';
+import { AzureKeyCredential } from '@azure/core-auth';
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const endpoint: string = process.env['VISION_ENDPOINT'] || '<your_endpoint>';
+const key: string = process.env['VISION_KEY'] || '<your_key>';
+const credential = new AzureKeyCredential(key);
+
+const client: ImageAnalysisClient = createImageAnalysisClient(endpoint, credential);
+
+const features: string[] = [
+  'Caption',
+  'DenseCaptions',
+  'Objects',
+  'People',
+  'Read',
+  'SmartCrops',
+  'Tags'
+];
+const imagePath: string = '../sample.jpg';
+
+async function analyzeImageFromFile(): Promise<void> {
+  const imageBuffer: Buffer = fs.readFileSync(imagePath);
+
+  const result = await client.path('/imageanalysis:analyze').post({
+    body: imageBuffer,
+    queryParameters: {
+      features: features,
+      'smartCrops-aspect-ratios': [0.9, 1.33]
+    },
+    contentType: 'application/octet-stream'
+  });
+
```

Comment:
you should be able to import `isUnexpected` helper which will remove the need to cast as `ImageAnalysisResultOutput`

```ts
if(isUnexpected(result)) {
   throw result.body,error;
}

// From here on result.body should be of type ImageAnalysisResultOutput without the need of casting
```